gulpfile.js

```diff
@@ -143,7 +143,7 @@ gulp.task('output:clean', () => del(['coverage']));
 
 gulp.task('clean:cleanExceptTests', () => del(['clean:vsix', 'out', '!out/test']));
 gulp.task('clean:vsix', () => del(['*.vsix']));
-gulp.task('clean:out', () => del(['out/**', '!out', '!out/client_renderer/**']));
+gulp.task('clean:out', () => del(['out/**', '!out', '!out/client_renderer/**', '!**/*nls.*.json']));
```

Comment:
One of the challenges I had was that the `*nls.*.json` files were missing in the final `.VSIX`. This change, and another change in the `.vscodeignore` were the solution.